projects/planner/src/app/mainmenu/mainmenu.component.html

```diff
@@ -27,6 +27,7 @@
                         <a class="dropdown-item" (click)="loadDefaults()">Reload defaults</a>
                     </div>
                 </li>
+                <a class="nav-link" routerLink="/diff">Compare Profiles</a>
```

Comment:
Dont publish link in the menu, until the feature is fully functional, so can publish anytime.